src/schema.rs

```diff
@@ -459,6 +470,18 @@ impl Schema {
                 "map" => Schema::parse_map(complex),
                 "fixed" => Schema::parse_fixed(complex),
                 "bytes" => Schema::parse_bytes(complex),
+                "int" => match complex.get("connect.name") {
+                    Some(name)
+                        if name == "io.debezium.time.Date"
+                            || name == "org.apache.kafka.connect.data.Date" =>
+                    {
+                        if name == "org.apache.kafka.connect.data.Date" {
+                            warn!("Using deprecated debezium date format")
+                        }
+                        Schema::parse_date()
```

Comment:
The function boundary here seems a bit weird! It seems like you might want this entire match arm in a function called `parse_int`, and just inline `parse_date` into it? (At the very least it seems like `parse_date` could be inlined here.)

Reply:
Ah, yeah, that would be more inline with the way the other things are being done, thanks!